I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 128, in script
    with slow_dissolve
  File "game/script.rpy", line 128, in <module>
    with slow_dissolve
NameError: name 'slow_dissolve' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 128, in script
    with slow_dissolve
  File "E:\renpy-8.2.1-sdk\renpy\ast.py", line 1354, in execute
    trans = renpy.python.py_eval(self.expr)
  File "E:\renpy-8.2.1-sdk\renpy\python.py", line 1209, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "E:\renpy-8.2.1-sdk\renpy\python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 128, in <module>
    with slow_dissolve
NameError: name 'slow_dissolve' is not defined

Windows-10-10.0.18363 AMD64
Ren'Py 8.3.2.24090902
MysticPages 1.0
Thu Nov 28 13:47:43 2024
